dgl_sparse/src/sparse_matrix.cc

```diff
@@ -122,6 +124,57 @@ c10::intrusive_ptr<SparseMatrix> SparseMatrix::FromDiag(
   return SparseMatrix::FromDiagPointer(diag, value, shape);
 }
 
+c10::intrusive_ptr<SparseMatrix> SparseMatrix::IndexSelect(
+    int64_t dim, torch::Tensor ids) {
+  auto id_array = TorchTensorToDGLArray(ids);
+  bool rowwise = dim == 0;
+  SparseFormat sparse_format;
+  if (rowwise) {
+    sparse_format = SparseFormat::kCSR;
+  } else {
+    sparse_format = SparseFormat::kCSC;
+  }
```

Comment:
```suggestion
  SparseFormat sparse_format = rowwise? SparseFormat::kCSR: SparseFormat::kCSC;
```

It is simply to write it in one line now.